lib/datasource/rubygems/get.ts

```diff
@@ -50,23 +50,47 @@ export async function getDependency(
     return null;
   }
 
-  const versions = (await fetch(dependency, registry, VERSIONS_PATH)) || [];
+  let versions = [];
+  let releases = [];
+  try {
+    versions = await fetch(dependency, registry, VERSIONS_PATH);
+  } catch (err) {
+    if (err.statusCode === 400 || err.statusCode === 404) {
+      logger.debug({ registry }, 'version endpoint errors or is not available');
+      logger.debug({ err });
+    } else {
+      throw err;
+    }
+  }
 
-  const releases = versions.map(
-    ({
-      number: version,
-      platform: rubyPlatform,
-      created_at: releaseTimestamp,
-      rubygems_version: rubygemsVersion,
-      ruby_version: rubyVersion,
-    }) => ({
-      version,
-      rubyPlatform,
-      releaseTimestamp,
-      rubygemsVersion,
-      rubyVersion,
-    })
-  );
+  if (versions.length === 0 && info.version) {
+    logger.warn('falling back to the version from the info endpoint');
+    releases = [
+      {
+        version: info.version,
+        rubyPlatform: info.platform,
+        releaseTimestamp: null,
+        rubyVersion: null,
+        rubygemsVersion: '\u003e= 0',
```

Comment:
What is the meaning of this value?

Reply:
it was the default value from the versions endpoint its basically >= 0

Reply:
but we can remove it I just wanted to match the signature from the "default" case

Reply:
Yes, remove it